modules/tensor_mechanics/src/materials/ComputeFiniteStrain.C

```diff
@@ -11,6 +11,7 @@ InputParameters validParams<ComputeFiniteStrain>()
 {
   InputParameters params = validParams<ComputeStrainBase>();
   params.addClassDescription("Compute a strain increment and rotation increment for finite strains.");
+  params.set<bool>("stateful_displacements") = true;
```

Comment:
The problem is that is that it's possible to set any parameter in our system through `Parameters::set<>()`, perhaps a design flaw of our inheritance scheme. then we lose all of the extended functionality of `InputParameters`. I'm not completely against this, I just know our users too well. It may be time to override the set method too to make sure people aren't bypassing our system... oh well